src/main/resources/application.conf

```diff
@@ -403,6 +403,9 @@ mantis {
       # Reduced block reward after Byzantium fork
       first-era-reduced-block-reward = "3000000000000000000"
 
+      # Reduced block reward after Constantinople fork
+      first-era-constantinople-reduced-block-reward = "2000000000000000000"
```

Comment:
You need to apply similar change in file `ethereumHF.conf` which ovverrides default values for eth chain